src/main/java/uk/gov/hmcts/reform/bsp/common/service/RegExpValidator.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.bsp.common.service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class RegExpValidator {
+
+    public static List<String> validateField(Map<String, String> data, String field, String validationRegex) {
+        List<String> validationMessages = new ArrayList<>();
+
+        if (data.containsKey(field)) {
+            String valueToValidate = data.get(field);
+            if (!valueToValidate.matches(validationRegex)) {
+                validationMessages.add(field + " is not in a valid format");
```

Comment:
Since this is reusable, perhaps add an option to pass a custom error message in format accepted by `String.format` ?

Reply:
I can use it, it's just copy&paste'd from other repo